view/frontend/web/js/view/payment/method-renderer/alma_monthly_payments.js

```diff
@@ -125,6 +123,22 @@ define(
                 return priceUtils.formatPrice(cents / 100, window.checkoutConfig.priceFormat);
             },
 
+            cartTotal: function () {
+                return priceUtils.formatPrice(window.checkoutConfig.quoteData.grand_total, window.checkoutConfig.priceFormat);
+            },
+
+            customerTotalCostAmount: function (cost) {
+                return priceUtils.formatPrice(cost / 100, window.checkoutConfig.priceFormat);
+            },
+
+            isAnnualInterestRate: function (rate) {
+                return ( rate!=null && rate > 0 ? true : false );
```

Comment:
add space between `rate != null`

and prefer display as yoda condition `null != rate` to avoid  unwanted assignation

Reply:
FIxed